Library/test-helpers/Dockerfile.registrar

```diff
@@ -1,9 +1,12 @@
 FROM quay.io/centos/centos:stream9
 COPY id_*.pub /root/
-COPY lime_con_start.sh /usr/local/bin/lime_con_start
```

Comment:
When you remove copying script for starting several processes/services like (sshd and keylime) the additional setup of SSH is useless.